sdk/keyvault/keyvault-secrets/package.json

```diff
@@ -28,6 +28,7 @@
     "LICENSE.txt",
     "README.md",
     "types/src/",
+    "types/keyvault-secrets.d.ts",
```

Comment:
In theory, you shouldn't need the /types/src directory, just the .d.ts file.

Reply:
done!